classes/task/ReviewReminder.php

```diff
@@ -95,7 +95,7 @@ public function executeActions(): bool
                     // need to check should we sent a BEFORE REVIEW REQUEST RESPONSE reminder
                     if ($numDaysBeforeReviewResponseReminderDue > 0 &&
                         $dateResponseDue->gt($currentDate) &&
-                        $dateResponseDue->diffInDays($currentDate) <= $numDaysBeforeReviewResponseReminderDue) {
+                        (int)abs($dateResponseDue->diffInDays($currentDate)) <= $numDaysBeforeReviewResponseReminderDue) {
```

Comment:
Hi @touhidurabir, why do we use abs() ? 
Can the difference be a negative number?
We also check $dateResponseDue->gt($currentDate), so it should not be a negative number, or?